docs/articles/call-settings.md

```diff
@@ -1,3 +1,61 @@
 # Call settings
 
-(Chris Bacon to write this)
\ No newline at end of file
+## Introduction
+
+All client libraries wrapping gRPC-based APIs (see [api-layers](api-layers.md))
```

Comment:
Change api-layers to "the API layers article" or something similar. Apologies if I suggested api-layers before as the name.